### ISSUE TITLE
Excel2ldr does not wipe out existing content in target register

### ISSUE THREAD

**Issue by jyucsiro** [CSIRO-LW-LD/ldrpyutils#32]
Changing existing content for a property and making multiple uploads to LDR does not wipe out existing content and replace with updated content. 

Would be good if the user had an option to specify whether to clear the existing content affected in the register or not.

**Comment by meganrwong** [comment]
yes - would be good. Just tried to change (delete) content in 'notes' section using tool but had to do via interface. Would be good if there was some user guidance doc on what is replaceable/changeable and what is not via tool (ie the if and type only?). Cheers :)
